hack/run-go-test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+PKG=$@
```

Comment:
maybe document all of the environment vars expected to be set. also what this file does